camunda-utils/Camunda/Camunda.py

```diff
@@ -31,7 +31,7 @@ class Camunda:
 
     def __init__(self):
         self.workspace_id = os.getenv("RPA_WORKSPACE_ID")
-        self.base_url = "http://localhost:36227"
+        self.base_url = os.getenv("RPA_BASE_URL", "http://127.0.0.1:36227")
```

Comment:
cc @Poundex , we discussed to allow configuration of the Base URL. I did not see an existing variable in https://github.com/camunda/rpa-worker/?tab=readme-ov-file#robot-job-execution , so I propose a new one following the existing schema

Reply:
Seems fine, I saw the change and have already started using this one